pkgs/by-name/on/onthespot/package.nix

```diff
@@ -45,18 +49,19 @@ python3.pkgs.buildPythonApplication rec {
     packaging
     pillow
     protobuf
-    pyqt5
-    pyqt5-sip
+    pyperclip
+    pyqt6
+    pyqt6-sip
     pyxdg
+    qt6.qtbase
     requests
-    setuptools
     show-in-file-manager
     urllib3
     zeroconf
   ];
 
   postInstall = ''
-    install -Dm444 $src/src/onthespot/resources/icon.png $out/share/icons/hicolor/256x256/apps/onthespot.png
+    install -Dm444 $src/src/onthespot/resources/icons/onthespot.png $out/share/icons/hicolor/256x256/apps/onthespot.png
   '';
 
   preFixup = ''
```

Comment:
Do we still need this preFixup? I just tested it without and it works fine (I did have to manually apply the PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION variable)